replicatedhq/ship
# Add PushMessageStep to StatusReceiver

What I Did
------------
Setup to address #433  (WIP)

How I Did it
------------
- Add daemonless versions of `lifecycle.Terraformer` and `tfplan.PlanConfirmer`
- Add `terraform/apply` and `terraform/skip` endpoints to navcycle
- Using `status` in `Progress.detail` to control "sub-steps" within a phase

How to verify it
------------
- Verify FE state changes through tests
- Verify BE changes by running `./bin/ship app --runbook integration/base/terraform/input/.ship/release.yml`
- A message will show with the terraform plan; however, further navigation is a WIP

Description for the Changelog
------------
N/A


Picture of a Boat (not required but encouraged)
------------
🛶
